src/workspace/resolver.rs

```diff
@@ -549,20 +552,18 @@ impl WorkspaceResolver {
         if let Some(path) = specifier.strip_prefix(&member.name) {
           if path.is_empty() || path.starts_with('/') {
             let path = path.strip_prefix('/').unwrap_or(path);
-            return self.resolve_workspace_jsr_pkg(
-              member,
-              JsrPackageReqReference::from_str(&format!(
-                "jsr:{}{}/{}",
-                member.name,
-                member
-                  .version
-                  .as_ref()
-                  .map(|v| format!("@^{}", v))
-                  .unwrap_or_else(String::new),
-                path
-              ))
-              .unwrap(),
-            );
+            let pkg_req_ref = JsrPackageReqReference::from_str(&format!(
+              "jsr:{}{}/{}",
+              member.name,
+              member
+                .version
+                .as_ref()
+                .map(|v| format!("@^{}", v))
+                .unwrap_or_else(String::new),
+              path
+            ))
+            .map_err(|e| MappedResolutionError::Other(e.to_string()))?;
```

Comment:
Should we use more concrete error variant like `Specifier` instead of Other? I thought be using `Specifier` but that requires updating import_map `SpecifierError` to support `PackageReqReferenceParseError`.

Or there is a better way to do it. 

what do you think @dsherret?

Reply:
The error message right now might be confusing for the user. I think we should surface invalid workspace member names as WorkspaceDiagnostics (https://github.com/denoland/deno_config/blob/5ab733afefba24b996594b96d24f730d061b007e/src/workspace/mod.rs#L124) and then silently ignore invalid workspace member names here.